merkle/src/store.rs

```diff
@@ -268,11 +437,19 @@ impl<E: Element> DiskStore<E> {
     }
 
     pub fn store_read_into(&self, start: usize, end: usize, buf: &mut [u8]) {
-        buf.copy_from_slice(&self.store_read_range(start, end));
+        assert_eq!(
+            self.file
+                .read_at(start as u64, buf)
+                .unwrap_or_else(|_| panic!(
+                    "failed to read {} bytes from file at offset {}",
+                    end - start, start
+                )),
+            end - start
+        );
     }
 
     pub fn store_copy_from_slice(&mut self, start: usize, slice: &[u8]) {
-        assert!(start + slice.len() <= self.store_size);
+        // assert!(start + slice.len() <= self.store_size);
```

Comment:
why is this commented out?

Reply:
Early testing.  It should not be commented out, thanks.